optaweb-employee-rostering-frontend/src/store/tenant/reducers.ts

```diff
@@ -45,4 +47,13 @@ const tenantReducer = (state = initialState, action: TenantAction): TenantData =
   }
 };
 
+export const connectionReducer = (state = initialConnectionState, action: ConnectAction): boolean => {
```

Comment:
Belongs to a separate file inside a connection `duck`. But the `store` is already breaking `ducks` organization on many places. Let's solve this outside of this PR.